linera-base/src/identifiers.rs

```diff
@@ -350,6 +351,8 @@ pub struct BytecodeId<Abi = (), Parameters = (), InstantiationArgument = ()> {
     pub contract_blob_hash: CryptoHash,
     /// The hash of the blob containing the service bytecode.
     pub service_blob_hash: CryptoHash,
```

Comment:
@afck @jvff Should we make the service blob optional?

Reply:
Maybe, if so it would have to be in a separate PR I think.

Reply:
Ok personally I was upvoting the enum where each VM variant can have a variable number of blobs. (I don't how easy it is to support, though)

Reply:
Currently, we have two bytecodes in the EVM for service and contract which are identical.
We have discussed merging contract/service for regular Linera applications. This may be an impetus for doing that.